### PR TITLE
added addon attachments

### PR DESCRIPTION
**Pull request by CAChristopher**

### CODE REVIEW

File: src/components/Addons/AttachAddon.jsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  }
+
+  getStepContent(stepIndex) {
+    switch (stepIndex) {
+      case 0:
+        return (
+          <div>
+            <Search className={'app-search'} label="App" data={util.filterName(this.state.apps)} handleSearch={this.handleSearch} searchText="" errorText={this.state.errorText} />
+            <p>
+              The application name or id to add to the pipeline. Ex. my-test-app-dev
```
Comment (murrayres) [inline src/components/Addons/AttachAddon.jsx]: Pipeline

File: src/components/Addons/AttachAddon.jsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+            <DropDownMenu className="addon-menu" value={this.state.addon} onChange={this.handleAddonChange}>
+              {this.getAddons()}
+            </DropDownMenu>
+            <p>
+              Select the plan for your addon (please only use larger plans for prod)
+            </p>
+          </div>
+        );
+      default:
+        return 'You\'re a long way from home sonny jim!';
```
Comment (murrayres) [inline src/components/Addons/AttachAddon.jsx]: Pardon

File: src/components/Addons/AttachAddon.jsx (hunk 1 above)
Comment (CAChristopher) [reply src/components/Addons/AttachAddon.jsx]: nice spot

File: src/components/Addons/AttachAddon.jsx (hunk 2 above)
Comment (CAChristopher) [reply src/components/Addons/AttachAddon.jsx]: this is an untravelable path, if someone sees this we need someone to talk about it or ask us

**Pull request merged**